packages/ui-view/src/View/index.js

```diff
@@ -57,7 +57,12 @@ category: components
 **/
 @deprecated('8.0.0', {
   focused: 'withFocusOutline',
-  visualDebug: 'withVisualDebug'
+  visualDebug: 'withVisualDebug',
+  background: `In version 8.0.0,
+    the value <default> for background will be changed to <primary>,
+    the value <light> for background will be changed to <secondary>,
+    the value <inverse> for background will be changed to <primary-inverse>.
+    Use these values instead.`
 })
```

Comment:
is this OK? If we put in here a deprecation notice, it will change in 9.0

Reply:
After this is merged into master, and master is again merged into next, we need to make cleanup on next to reverse these.